libs/barista-components/autocomplete/src/autocomplete-trigger.ts

```diff
@@ -297,6 +316,16 @@ export class DtAutocompleteTrigger<T>
           }
         });
     }
+
+    if (_optionConfig) {
+      this._optionHeight = _optionConfig.height;
+      this._panelMaxHeight = Math.ceil(
+        _optionConfig.height * OPTION_TO_PANEL_HEIGHT_RATIO,
+      );
```

Comment:
What if the option height is really large - lets say 100px - this is not capped to any value. 
We need to cap this - I'd rather have a logic that tries to fit as many options as possible into the AUTOCOMPLETE_PANEL_MAX_HEIGHT, and in the case that the optionheight is a multiple of the AUTOCOMPLETE_PANEL_MAX_HEIGHT decrease the panel height by half an option height.
This would not let the overlay grow indefinitely and solve the problem that if the height is a multiple of the option height - it looks like all options are shown.

Reply:
Good idea, I'll do that!